common/defs/bitcoin/firo.json

```diff
@@ -1,17 +1,17 @@
 {
-  "coin_name": "Zcoin",
-  "coin_shortcut": "XZC",
-  "coin_label": "Zcoin",
-  "website": "https://zcoin.io",
-  "github": "https://github.com/zcoinofficial/zcoin",
+  "coin_name": "Firo",
+  "coin_shortcut": "FIRO",
+  "coin_label": "Firo",
+  "website": "https://firo.org",
+  "github": "https://github.com/firoorg/firo",
   "maintainer": "Andrey Bezrukov <andreyvbezrukov@gmail.com>",
   "curve_name": "secp256k1",
   "decimals": 8,
   "address_type": 82,
   "address_type_p2sh": 7,
   "maxfee_kb": 640000000,
   "minfee_kb": 0,
-  "signed_message_header": "Zcoin Signed Message:\n",
+  "signed_message_header": "Firo Signed Message:\n",
```

Comment:
Don't change this row - it seems the magic stayed the same: https://github.com/firoorg/firo/blob/b39fa5fba3f0554b8502ee115fd2eb6a062a8471/src/validation.cpp#L144

Reply:
Thanks!